timeout/timeout.c

```diff
@@ -78,7 +78,7 @@ int main(int argc, char *argv[]) {
 	    ecmd = 0, /* Command exit code. */
 	    eprog = 0; /* Program exit code. */
 	
-	char **commandv = { (char)0 },
+	char **commandv = { (char **)"" },
```

Comment:
This is simply not valid does this even compile?

Reply:
Everything is wrong please don't code on marijuana anymore

Reply:
> Everything is wrong please don't code on marijuana anymore

Maybe my coffee was spoiled...
How the hell are we supposed to initialize a character array-array? I thought ``{}`` already implied that we are wanting to initialize an array.

Reply:
> This is simply not valid does this even compile?

It does, without warnings.